device/globals/global_vm.gd

```diff
@@ -306,7 +316,12 @@ func sched_event(time, obj, event):
 	event_queue.push_back([time, obj, event])
 
 func get_global(name):
-	return (name in globals) && globals[name]
+	# If no value or boolean, return boolean for backwards compatibility
+	if not name in globals:
+		return false
+	if typeof(globals[name]) == TYPE_BOOL:
+		return !!globals[name]
```

Comment:
It's probably safer to treat all globals as strings going forward and compare the string to "true"+"false".